src/app/app.config.ts

```diff
@@ -27,7 +27,7 @@ export class AppConfig extends AbstractAppConfig {
         .catch((error: any): any => {
           console.error(`Configuration ${configUrl} could not be read`, error);
           reject();
-          return Observable.throw(error.json().error || 'Server error');
+          return throwError(error.json().error || 'Server error');
```

Comment:
That change is not related to the POC and is a good change to bring in the code base

Reply:
Sure will bring it in.